app.py

```diff
@@ -13,13 +12,18 @@ def create_app(config_name):
     app.config.from_object(config[config_name])
     config[config_name].init_app(app)
     
-    app.add_url_rule(
-        '/mrm',
-        view_func=GraphQLView.as_view(
+    def graphql_view():
```

Comment:
I am against authenticating routes in graphQL, remember we only have one route, and there are things we would need no authentication for, why not just go the route of authorization